students/christopher_gantt/lesson04/mailroom.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+
+DONOR_DB = {"Wassily Kandinsky": [43928.13, 131.34, 1928.0],
+            "Jasper Johns": [3134.43, 153.34],
+            "Mark Rothko": [135353.33],
+            "Richard Serra": [153757.87, 28457.12, 1293451.0],
+            "Yves Tanguy": [1534.23, 2542.19]}
```

Comment:
:thumbsup:

not that this should be `donor_db` - all caps is used for constants (things that are immutable)

Reply:
Thank you! I thought it was used for all global variables.